hack/generator/pkg/astmodel/testdata/ConvertBetweenEnumAndBaseType.golden

```diff
@@ -0,0 +1,30 @@
+// Code generated by k8s-infra. DO NOT EDIT.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package vCurrent
+
+import "github.com/Azure/k8s-infra/hack/generated/_apis/Verification/vNext"
+
+type Person struct {
+	sku string `json:"sku"`
+}
+
+// ConvertFromVNext populates our Person from the provided source Person
+func (person Person) ConvertFromVNext(source *vNext.Person) error {
+
+	// sku
+	person.sku = string(source.sku)
+
+	// No error
+	return nil
+}
+
+// ConvertToVNext populates the provided destination Person from our Person
+func (person Person) ConvertToVNext(destination *vNext.Person) error {
+
+	// sku
+	destination.sku = Bucket(person.sku)
```

Comment:
Should `Bucket` be `vNext.Bucket`?

Reply:
Yes - the test was using the wrong enum when creating the property for testing (was using one from the vCurrent package instead of the vNext package). Fixed.